### PR TITLE
Added jax.numpy.positive fixes #8426

### LINKED ISSUE #8426: Add `jax.numpy`  mathematical functions to Jax frontend
These functions can be found in [`jax.numpy`](https://jax.readthedocs.io/en/latest/jax.numpy.html) namespace

- [x] #4884
- [x] #4915
- [x] #4916
- [ ] angle
- [x] #5248
- [x] #5249
- [x] #5250
- [x] #5251
- [x] #5369
- [x] #5371
- [x] #6895
- [ ] around
- [ ] cbrt
- [x] #6326
- [x] #4957
- [ ] conj
- [ ] conjugate
- [ ] convolve
- [ ] copysign
- [x] cos
- [ ] cosh
- [x] #6467
- [ ] #7477
- [ ] diff
- [x] #7771
- [ ] divmod
- [x] #4946
- [ ] ediff1d
- [ ] einsum_path
- [ ] exp
- [x] #6471
- [ ] expm1
- [ ] fabs
- [ ] fix
- [x] #6321
- [x] #5691
- [ ] floor_divide
- [x] #5705
- [ ] fmin
- [ ] fmod
- [ ] frexp
- [x] #6532
- [ ] gradient
- [x] #6458
- [ ] hypot
- [x] #6561
- [ ] imag
- [ ] inner
- [ ] interp
- [x] #6584
- [x] #6599
- [ ] ldexp
- [ ] log
- [x] #7277
- [x] #5895
- [ ] #5896
- [x] #7286
- [x] #6602
- [ ] matmul
- [x] #7049
- [x] #6901 
- [x] #4953
- [ ] #6900
- [ ] mod
- [ ] modf
- [x] #6899
- [ ] nan_to_num
- [ ] nanprod
- [x] #7711
- [ ] nextafter
- [ ] outer
- [ ] poly
- [ ] polyadd
- [ ] polyder
- [ ] polydiv
- [ ] polyfit
- [ ] polyint
- [ ] polymul
- [ ] polysub
- [ ] polyval
- [ ] positive
- [x] #6082
- [ ] prod
- [ ] product
- [x] #7749
- [ ] radians
- [ ] real
- [ ] reciprocal
- [ ] remainder
- [ ] rint
- [ ] roots
- [ ] round
- [ ] round_
- [ ] sign
- [ ] signbit
- [x] #6896
- [ ] sinc
- [x] #6897
- [x] #7854
- [ ] square
- [ ] #5300
- [x] #6317
- [x] #7165
- [x] #7166
- [x] #7755
- [ ] trace
- [x] #7855
- [x] #6618
- [x] #7776
- [x] #6330
- [ ] unwrap
- [ ] vdot

### LINKED ISSUE #10367: positive

### PR DESCRIPTION
**Pull request by abhijeetsharma200**
Close #10367

### CODE REVIEW

**Comment by abhijeetsharma200** [comment]
@fspyridakos I would appreciate some help in figuring why the tests are failing.

**Comment by fspyridakos** [comment]
LGTM! The tests that are failing are unrelated to your PR. Thanks for your contribution!

**Comment by fspyridakos** [comment]
LGTM! The tests that are failing are unrelated to your PR. Thanks for your contribution! Just know for next time if you want to work on a function in a ToDo list, you should open an issue first and then comment on the ToDo as explained here: https://lets-unify.ai/ivy/contributing/the_basics.html#todo-list-issues . This is to reserve the function for yourself and helps us know who is working on what. I did this for you now but for the future please reference that guide. Thanks again!

**Comment by abhijeetsharma200** [comment]
I see, thank you for your advice and I will keep this in mind for my next issue :)

Looking forward to making more contributions!
